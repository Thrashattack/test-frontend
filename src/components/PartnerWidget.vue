<template>
  <div class="PartnerWidget d-flex flex-column justify-content-center align-items-center text-center">
    <img :src="img" :alt="img" height="99px" width="99px">
    <span>{{name}}</span>
  </div>
</template>

<script>
export default {
  name: "PartnerWidget",
  props: {
    img: String,
    name: String
  }
};
</script>

<style lang="scss" scoped>
.PartnerWidget {
  background: #FFFFFF;
  border: 1px solid #ECECEC;
  box-sizing: border-box;
  span {
    margin-top: 18.55px;
    font-family: Encode Sans;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    line-height: 19px;
    text-align: center;
    letter-spacing: 0.503704px;
    color: #3F3F3F;
  }
}
</style>
