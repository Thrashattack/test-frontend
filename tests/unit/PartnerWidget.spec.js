import { shallowMount } from "@vue/test-utils";
import PartnerWidget from "@/components/PartnerWidget.vue";

describe("PartnerWidget.vue", () => {
  it("renders the component", () => {
    const wrapper = shallowMount(PartnerWidget);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
