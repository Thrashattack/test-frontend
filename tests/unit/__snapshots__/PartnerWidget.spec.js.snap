// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PartnerWidget.vue renders the component 1`] = `<div class="PartnerWidget d-flex flex-column justify-content-center align-items-center text-center"><img height="99px" width="99px"> <span></span></div>`;
